### PR TITLE
Adjust Typings to reflect actual values

### PR DESCRIPTION
**Pull request by dmiller9911**
* onChange returns a SyntheticNative event with CardTokenParams 'number', 'expMonth',  'expYear', 'cvc'.
* expMonth and expYear are now typed as numbers for the change events.
* Fixed some property defs to allow for better module augmentation.
* Styling (prettier plugin). I left the formatting (mainly quotes) since it matches the lint guidelines from tipsi.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -227,12 +227,7 @@ export class Stripe {
 
 export interface PaymentCardTextFieldNativeEvent {
   valid: boolean;
-  params: {
-    number: string;
-    expMonth: string;
-    expYear: string;
-    cvc: string;
-  };
+  params: Pick<CardTokenParams, 'number' | 'expMonth' | 'expYear' | 'cvc'>;
```
Comment (dmiller9911) [inline index.d.ts:230]: Fix is here for `onParamsChange`. The `expMonth: string;` and `expYear: string;` were wrong they are actually numbers, and match the setup of `CardTokenParams`

File: index.d.ts [hunk 2]
```diff
@@ -241,13 +236,14 @@ export interface PaymentCardTextFieldCommonProps {
   cvcPlaceholder?: string;
   disabled?: boolean;
 
-  onChange?: (params: PaymentCardTextFieldNativeEvent) => void;
+  onChange?(
+    event: ReactNative.NativeSyntheticEvent<PaymentCardTextFieldNativeEvent>
```
Comment (dmiller9911) [inline index.d.ts:240]: This is actually a Syntentic event. The event data is under `event.nativeEvent`

File: index.d.ts [hunk 3]
```diff
@@ -241,13 +236,14 @@ export interface PaymentCardTextFieldCommonProps {
...
+  ): void;
 
-  /** @deprecated */
```
Comment (dmiller9911) [inline index.d.ts:246]: I could not find anywhere that documents this as deprecated. It is actually what is used in their example in their docs so I think it is safe to remove the deprecated flag.

File: index.d.ts [hunk 4]
```diff
@@ -241,13 +236,14 @@ export interface PaymentCardTextFieldCommonProps {
...
-  onParamsChange?: (
+  onParamsChange?(
```
Comment (dmiller9911) [inline index.d.ts:243]: changes this to not use the `=>` syntax. It doesn't really allow module augmentation as well as the syntax I changed it to (not sure the name for it)

File: index.d.ts [hunk 5]
```diff
@@ -268,15 +264,15 @@ export interface PaymentCardTextFieldAndroidProps {
...
+  Omit<ReactNative.ViewProps, 'style'> &
     PaymentCardTextFieldCommonProps &
     PaymentCardTextFieldIOSProps &
     PaymentCardTextFieldAndroidProps
 > {
-  isFocused: () => boolean;
-  focus: () => void;
-  blur: () => void;
-  setParams: (params: CardTokenParams) => void;
+  isFocused(): boolean;
```
Comment (dmiller9911) [inline index.d.ts:272]: Changed these as well to improve module augmentation support.

**Review by 0xch4z — APPROVED** [review]

**Pull request merged**